testing/libbson/APKBUILD

```diff
@@ -1,35 +1,41 @@
 # Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
 pkgname=libbson
 pkgver=1.9.2
-pkgrel=1
+pkgrel=2
 pkgdesc="routines related to building, parsing, and iterating BSON documents"
 url="https://github.com/mongodb/libbson"
 arch="all"
 license="Apache-2.0"
-makedepends="$depends_dev autoconf automake libtool"
+depends=
+depends_dev=
```

Comment:
Why did you add empty variables?! Could you please stop doing non-sense changes in abuilds that just increase verbosity?!